modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/configuring-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-configuring-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure {PM-kepler} to use Redfish as the source for running or hosting containers. {PM-kepler} can then monitor the power usage of these containers.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and click the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*. If you already have a {PM-kepler} instance created, click *Edit Kepler*.
+
+. Configure `.spec.exporter.redfish` of the {PM-kepler} instance by specifying the mandatory `secretRef` field. You can also configure the optional `probeInterval` and `skipSSLVerify` fields to meet your needs.
++
+.Example {PM-kepler} instance
+[source,yaml]
+----
+apiVersion: kepler.system.sustainable.computing.io/v1alpha1
+kind: Kepler
+metadata:
+  name: kepler
+spec:
+  exporter:
+    deployment:
+# ... 
+    redfish:
+      secretRef: <secret name> required <1>
+      probeInterval: 60s <2>
+      skipSSLVerify: false <3>
+# ...
+----
+<1> Required: Specifies the name of the secret that contains the credentials for accessing the Redfish server.
+<2> Optional: Controls the frequency at which the power information is queried from Redfish. The default value is `60s`.
+<3> Optional: Controls if {PM-kepler} skips verifying the Redfish server certificate. The default value is `false`.
++
+[NOTE]
+====
+After {PM-kepler} is deployed, the `openshift-power-monitoring` namespace is created.
+====
+. Create the `redfish.csv` file with the following data format:
++
+[source,csv]
+----
+<your_kubelet_node_name>,<redfish_username>,<redfish_password>,\https://<redfish_ip_or_hostname>/
```

Comment:
Sorry, I accidentally left the `\` solution in when I was copy-pasting around in my original comment. The `\` is only necessary when it's in monospace/backticks in-line of a sentence, but not when it's a full codeblock like this:
```suggestion
<your_kubelet_node_name>,<redfish_username>,<redfish_password>,https://<redfish_ip_or_hostname>/
```